dscreator/main.py

```diff
@@ -211,5 +211,38 @@ def nrt_color_fantasy(max_time_slice: int = 24, stop_after_n_files: int = -1, ac
     runner.start()
 
 
+@app.command()
+def ramses_color_fantasy(max_time_slice: int = 24, stop_after_n_files: int = -1, acdd: ACDDOptions = "no"):
+    """Build nrt color fantasy dataset from data in tsb"""
+
+    logging.info("Exporting Ramses FA dataset")
+    trajectory_extractor = ferrybox.extractor.SpectraExtractor(
+        create_engine(SETTINGS.tsb_connection_str),
+        variable_codes=["rrs_344", "rrs_452", "rrs_671"],
```

Comment:
We will need the whole spectra according to Therese, from 323 nm to 899 (in 3 nm steps)